spec/gitlab/client/projects_spec.rb

```diff
@@ -925,4 +925,15 @@
       expect(@custom_attributes.first.username).to eq 'gitlab+deploy-token-93'
     end
   end
+
+  describe '.project_languages' do
+    before do
+      stub_get('/projects/2/languages', 'project_languages')
```

Comment:
There is no fixture with this name, so the test won't pass.

Reply:
Fixed